recipes/Cbc/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+
+UNAME="$(uname)"
+export CFLAGS="${CFLAGS} -O3"
+export CXXFLAGS="${CXXFLAGS} -O3"
+export CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++11}"
+
+# if [ "${UNAME}" == "Linux" ]; then
+#   export FLIBS="-static-libgcc -Wl,-Bstatic,-lstdc++,-Bdynamic -lm"
+# fi
+
+# Use only 1 thread with OpenBLAS to avoid timeouts on CIs.
+# This should have no other affect on the build. A user
+# should still be able to set this (or not) to a different
+# value at run-time to get the expected amount of parallelism.
+export OPENBLAS_NUM_THREADS=1
+
+WITH_BLAS_LIB="-L${PREFIX}/lib -lblas"
+WITH_LAPACK_LIB="-L${PREFIX}/lib -llapack"
+
+./configure --prefix="${PREFIX}" --exec-prefix="${PREFIX}" \
```

Comment:
This may be a little confusing, but in the current releases, there are actually two `configure` scripts for each project. The one in the root directory used to play the role of being a "meta-configure" to find all the dependencies and recursively build each of them. This allowed monolithic builds from a single checkout using a single build command (this is what was done in the non-split build that we are replacing). This goes back to the way `svn` worked with `externals` before we moved to `git`. 

In a split build like we're doing here, the root `configure` script is superfluous. All it does is find the project itself and run it's `configure`. So you can actually just run the `configure` script in the Cbc subdirectory directly, which simplifies things and will make the build a lot faster. This is the case for all the projects.  

Of course, this will all change when the current development version move to release. We eliminated the root directory now and don't support monolithic builds anymore. This is the reason the split build will be required in the future.

So you could leave it as is, knowing that this will mean the current recipes will just work when Cbc 3.0 comes out. Or you could change the command to `Cbc/configure`. I don't think there are any serious side-effects, other than slowing down the build,  to leaving it as is. It does make the output of configure a little hard to parse, as there are two configurations smashed together.